src/Interface/Modules/Render/ViewScene.h

```diff
@@ -124,6 +125,7 @@ namespace SCIRun {
       void pullCameraRotation();
       void pullCameraLookAt();
       void pullCameraDistance();
+      glm::quat stringToQuat(std::string &s);
```

Comment:
```suggestion
      glm::quat stringToQuat(const std::string &s);
```

Move this function to a new header/cc file, as a global function or in a new class, called ViewSceneUtility or the like.